### DISCUSSION TITLE
What's new?

### DISCUSSION THREAD

**Discussion by PaulWilliamWoods** [freeplane/freeplane#1038] [General]
I'm trying to find a list or a set of lists which detail what is being worked on in upcoming releases. There are, for example, some seriously interesting new formatting options coming up in 1.11, perhaps so that I will be able to create a fishbone-like structure. However, I only found out about that from seeing a picture and I found that picture by search. Is there a place here on github with that sort of 'works in progress' detail?

**Comment by EdoFro** [comment]
Maybe the '**History**' text file in [Sourceforge preview repository](https://sourceforge.net/projects/freeplane/files/freeplane%20preview/) can help.

**Comment by PaulWilliamWoods** [comment]
Thank you; yes, that's not bad. The history file there does give me a bit
of what I was looking for. Is there also a place where release date targets
are written?
